### ISSUE TITLE
accessibility problem

### ISSUE THREAD

**Issue by GaryMunoz** [microsoft/vscode#214698]
Type: <b>Feature Request</b>

good day, I have problems with the latest update of Visual Code in accessibility, it does not indicate errors with the sound alert when a line of code has an error, in the previous version and subsequent versions it did do this and the screen reader also indicated the ERROR, I am attentive, thanks, I hope you can solve it, it is very helpful for people with visual disabilities.

VS Code version: Code 1.90.0 (89de5a8d4d6205e5b11647eb6a74844ca23d2573, 2024-06-04T19:33:54.889Z)
OS version: Windows_NT x64 10.0.19045
Modes:

**Comment by meganrogge** [comment]
I see this in the console:

```
 Cannot read properties of undefined (reading 'sound'): TypeError: Cannot read properties of undefined (reading 'sound')
```

**Comment by bpasero** [comment]
See also https://github.com/microsoft/vscode/issues/214714 and maybe https://github.com/microsoft/vscode/issues/214717

**Comment by GaryMunoz** [comment]
Hello, question, was the issue resolved? Since I updated to 1.90 again and I'm still having the problem, many thanks.

**Comment by benibenj** [comment]
@GaryMunoz we will be releasing a fix towards end of the week

**Comment by GaryMunoz** [comment]
Very grateful, thanks.
